File: CS/WebApplication1/ReportService1.svc.cs
using DevExpress.Data.Utils.ServiceModel;
using DevExpress.XtraReports.Data;
using DevExpress.XtraReports.Service;
using System.Configuration;
// ...

namespace WebApplication1 {
    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "ReportService1" in code, svc and config file together.
    [SilverlightFaultBehavior]
    public class ReportService1 : DevExpress.XtraReports.Service.ReportService {
        static ReportService1() {
            DataProviderRepository.Current.Register(
                "nwind", 
                "Northwind", 
                ConfigurationManager.ConnectionStrings["nwindConnectionString"].ConnectionString);
        }
    }
  
}

File: VB/WebApplication1/ReportService1.svc.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Data.Utils.ServiceModel
Imports DevExpress.XtraReports.Data
Imports DevExpress.XtraReports.Service
Imports System.Configuration
' ...

Namespace WebApplication1
	' NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "ReportService1" in code, svc and config file together.
	<SilverlightFaultBehavior> _
	Public Class ReportService1
		Inherits DevExpress.XtraReports.Service.ReportService
		Shared Sub New()
			DataProviderRepository.Current.Register("nwind", "Northwind", ConfigurationManager.ConnectionStrings("nwindConnectionString").ConnectionString)
		End Sub
	End Class

End Namespace
